examples/db/cluster/with_scheduled_pitr_backup/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "vkcs_db_cluster" "pg_with_backup" {
+  name = "pg-with-backup"
```

Comment:
we use tf-example suffix in object names to let a user know the origin of the object (if he forgets to call tf destroy)